README.md

```diff
@@ -252,170 +244,187 @@ Assuming that you've successfully passed the "Connect" tests as described above,
+   Please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
+
+   Also, there are board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [quick start](https://cloud.mbed.com/quick-start).
 
-  Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [Quick-Start](https://cloud.mbed.com/quick-start).
+1. Set a global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine valid for the account your device will connect to. For example:
 
-2. Set a global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine valid for the account that your device will connect to. For example:
     ```
     $ mbed config -G CLOUD_SDK_API_KEY <API_KEY>
     ```
 
     For instructions on how to generate an API key, please [see the documentation](https://cloud.mbed.com/docs/current/integrate-web-app/api-keys.html#generating-an-api-key).
 
-3. Initialize your Pelion DM credentials (once per project):
+1. Initialize your Pelion DM credentials (once per project):
+
     ```
     $ mbed dm init -d "<your company name.com>" --model-name "<product model identifier>"
     ```
 
-    This will create your private/public key pair and also initialize various .c files with these credentials, so you can use Connect and (firmware) Update device management features.
+    This creates your private and public key pair and also initializes various `.c` files with these credentials, so you can use Connect and (firmware) Update device management features.
 
-4. Remove the `main.cpp` application from the project, or ensure the content of the file is wrapped with `#ifndef MBED_TEST_MODE`.
+1. Remove the `main.cpp` application from the project, or ensure the content of the file is wrapped with `#ifndef MBED_TEST_MODE`.
  
-5. Compile the tests with the `MBED_TEST_MODE` compilation flag.
+1. Compile the tests with the `MBED_TEST_MODE` compilation flag.
 
     ```
     $ mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* -DMBED_TEST_MODE --compile
     ```
 
-5. Run the tests from the application directory:
+1. Run the tests from the application directory:
 
     ```
     $ mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* --run -v
     ```
 
 ### Troubleshooting
-Below are a list of common issues and fixes.
+
+Below are common issues and fixes.
```

Comment:
Would it make sense to list these in a table or as bullets instead of as individual sections with their own headings?